### DISCUSSION TITLE
adding extensions?

### DISCUSSION THREAD

**Discussion by solanuscasey** [3kh0/ext-remover#1189] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
are there any working ways to add extensions? there appear to be plenty of ways to remove them but I didn't check each one to see if they can also add extensions. for context dev mode is not enabled, Linux is not enabled, javascript:// is blocked(policy), crosh does not work either. I don't know that there are any solutions to this, but let me know if you know of a method.

**Comment by S-PScripts** [comment]
[ExtensionInstallAllowlist]

[ExtensionInstallBlocklist]
[ExtensionInstallForcelist]

i think these policies will probably make it impossible to get extensions but thats just me



edit: this exists https://github.com/S-PScripts/chromebook-utilities/blob/main/Exploits/Lacros%20Bypass%20(Install%20ext.)%20(v96) but i doubt its useful for you. also there is a /ExtensionInstallAllowlist Opener
